tests/e2e/graphics/test-cases/initial-options/price-line-source-default.js

```diff
@@ -0,0 +1,24 @@
+function generateData() {
+	var res = [];
+	var time = new Date(Date.UTC(2018, 0, 1, 0, 0, 0, 0));
+	for (var i = 0; i < 500; ++i) {
+		res.push({
+			time: time.getTime() / 1000,
+			value: i,
+		});
+
+		time.setUTCDate(time.getUTCDate() + 1);
+	}
+	return res;
+}
+
+// eslint-disable-next-line no-unused-vars
+function runTestCase(container) {
+	var chart = LightweightCharts.createChart(container);
+
+	var series = chart.addLineSeries();
+
+	series.setData(generateData());
+
+	chart.timeScale().scrollToPosition(-100);
```

Comment:
```suggestion
	chart.timeScale().scrollToPosition(-10);
```

Currently we scroll a far away of last bar so we don't see price line on the screenshots. It'd be better to see it to handle whether something is changed there.